From f65bf988589ba9ec5bf9c996b66609af473be9db Mon Sep 17 00:00:00 2001
From: Anton Kikin <a.kikin@tano-systems.com>
Date: Wed, 30 Oct 2019 00:15:26 +0300
Subject: [PATCH 1/3] cli: fix type displaying for BLOBMSG_TYPE_DOUBLE
 arguments

Display BLOBMSG_TYPE_DOUBLE type ubus methods arguments
as "Double" instead of "(unknown)".

Signed-off-by: Anton Kikin <a.kikin@tano-systems.com>
---
 cli.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cli.c b/cli.c
index f566279..f9887e1 100644
--- a/cli.c
+++ b/cli.c
@@ -45,6 +45,7 @@ static const char *format_type(void *priv, struct blob_attr *attr)
 		[BLOBMSG_TYPE_STRING] = "\"String\"",
 		[BLOBMSG_TYPE_ARRAY] = "\"Array\"",
 		[BLOBMSG_TYPE_TABLE] = "\"Table\"",
+		[BLOBMSG_TYPE_DOUBLE] = "\"Double\"",
 	};
 	const char *type = NULL;
 	size_t typeid;
-- 
2.16.2.windows.1

